### PR TITLE
DEV008 create controller for place model

### PR DESCRIPTION
**Pull request by AS-AlStar**
Card DEV008 Create controller for place model (https://trello.com/c/mDLeexLV)
Create places controller, add serializer places. 
Add spec files for places controller.

### CODE REVIEW

File: app/controllers/places_controller.rb [hunk 1]
```diff
@@ -0,0 +1,41 @@
+class PlacesController < ApplicationController
+  def index
+    @places = Place.all
+    render json: @places
+  end
+
+  def show
+    @place = Place.find(params[:id])
```
Comment (andreydro) [inline app/controllers/places_controller.rb]: `@place = Place.find(params[:id])` repeat a lot. Move it to `before_action` and set it for actions where this line is used

File: spec/controllers/api/places_controller_spec.rb [hunk 2]
```diff
@@ -0,0 +1,19 @@
+RSpec.describe Api::PlacesController do
+
+  describe "GET #index" do
+    before do
+      create(:place)
```
Comment (andreydro) [inline spec/controllers/api/places_controller_spec.rb]: move `create(:place)` to `let!`

File: app/controllers/api/places_controller.rb [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+      render json: @places
+    end
+
+    def show
+      render json: @place
+    end
+
+    def create
+      @place = Place.new(place_params)
+      if @place.save
```
Comment (longbeardmen) [inline app/controllers/api/places_controller.rb]: `.save!`

File: app/controllers/api/places_controller.rb [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+        render json: @place, status: :created
+      else
+        render json: @place.errors, status: :unprocessable_entity
+      end
+    end
+
+    def update
+      if @place.update(place_params)
```
Comment (longbeardmen) [inline app/controllers/api/places_controller.rb]: `.update!`

File: app/controllers/api/places_controller.rb [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+
+    def destroy
+      @place.destroy
+      head :no_content
+    end
+
+    private
+
+    def set_place
+      @place = Place.find(params[:id])
```
Comment (longbeardmen) [inline app/controllers/api/places_controller.rb]: `params[:id]` should be permitted.
update `place_params` and work only with them

File: spec/controllers/api/places_controller_spec.rb [hunk 6]
```diff
@@ -0,0 +1,19 @@
...
+      get :index
+    end
+
+    it "returns http success" do
+      expect(response).to have_http_status(:success)
+    end
+
+    it "JSON body response contains expected recipe attributes" do
+      json_response = JSON.parse(response.body)
```
Comment (longbeardmen) [inline spec/controllers/api/places_controller_spec.rb]: it should be defined with `let`

File: spec/factories/places_factories.rb [hunk 7]
```diff
@@ -0,0 +1,10 @@
+FactoryBot.define do
+  factory :place do
+    name { 'Street' }
+    latitude { 1111.1111 }
+    longitude { 2222.2222 }
```
Comment (longbeardmen) [inline spec/factories/places_factories.rb]: provide proper coords for place

File: app/controllers/api/places_controller.rb [hunk 8]
```diff
@@ -0,0 +1,47 @@
...
+    def update
+      if @place.update(place_params)
+        render json: @place, status: :ok
+      else
+        render json: @place.errors, status: :unprocessable_entity
+      end
+    end
+
+    def destroy
+      @place.destroy
```
Comment (longbeardmen) [inline app/controllers/api/places_controller.rb]: what if something gonna fail here?

**Pull request merged**